ckanext/bcgov/templates/package/read.html

```diff
@@ -119,7 +119,7 @@
   {% block package_additional_info %}
     <h3>Additional Information</h3>
     <div id="additional-information">
-      {% if pkg.object_name and pkg.resources and pkg.resources[0].resource_storage_location == 'BCGW Data Store' %}
+      {% if pkg.object_name and pkg.resources %}
```

Comment:
yes, it was quick and easy. I'm just getting set up with a local dev environment so I can make and test the other little changes.